ribs/_docstrings.py

```diff
@@ -86,12 +86,12 @@ def from_nested_components(cls, **kwargs):
     metadata (numpy.ndarray): 1D object array containing a metadata
         object for each solution.
     """,
-                 add_statuses="""
-    add_statuses (numpy.ndarray): 1D array of :class:`ribs.archive.AddStatus`
+                 status_batch="""
+    status_batch (numpy.ndarray): 1D array of :class:`ribs.archive.AddStatus`
```

Comment:
say it's an array of "integer statuses" since I don't return AddStatus from the batched addition (you can't make a NumPy array that has Python enums)